nautobot/core/apps/__init__.py

```diff
@@ -24,6 +26,22 @@
 logger = logging.getLogger(__name__)
 registry["nav_menu"] = dict({tab: {"groups": {}, "permissions": set()} for tab in MENU_TABS})
 registry["homepage_layout"] = {"panels": {}}
+registry["system_jobs"] = []
+
+
+def register_jobs(class_list):
```

Comment:
Can this be `register_system_jobs` to avoid confusion with `nautobot.core.celery.register_jobs`?

Reply:
Hopefully this can go away by making the `class_path` property of `BaseJob` smarter. Since this doesn't do much more than marking a job as a system job instead of an unknown/plugin job.

Reply:
Is this registry necessary? What's the difference between a job and a system job?